opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -153,7 +153,7 @@ namespace Opm
         {}
 
         /** \copydoc Opm::OutputWriter::writeInit */
-        void writeInit(const SimulatorTimerInterface& /* timer */)
+        void writeInit(const SimulatorTimerInterface& /* timer */, const NNC& nnc = NNC())
```

Comment:
I would prefer if the `nnc` argument was not optional; the code in question is going through a lot of churn at the moment - and I think the risk of future fallouts increases with the use of optional. When things have stabilized optional might be fine.